cli/features/tutorial.feature

```diff
@@ -159,14 +160,17 @@ Feature: Following the tutorial
 
       dependencies:
         mongo:
-          version: '3.4.0'
-          docker_flags:
-            volume: '-v {{EXO_DATA_PATH}}:/data/db'
-            online_text: 'waiting for connections'
-            port: '-p 27017:27017'
+          dev:
```

Comment:
I'm still torn whether we should move the definition of dependencies to `application.yml` or leave it here. 

Ideally a service just says: "I need MongoDB version 3.4.0`. Or even better, it says "I need MongoDB version `3.x.x`. Then Exosphere collects all the requirements of all services and boots the smallest possible set of MongoDB image that satisfies all requirements of all services.